assets/js/modules/analytics-4/datastore/settings.test.js

```diff
@@ -521,27 +571,266 @@ describe( 'modules/analytics-4 settings', () => {
+					} );
+
+				const validSettings = {
+					accountID: fixtures.createProperty._accountID,
+					propertyID: fixtures.createProperty._id,
+					webDataStreamID: fixtures.createWebDataStream._id,
+				};
+
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setSettings( validSettings );
+
+				fetchMock.postOnce( settingsEndpoint, {
+					body: validSettings,
+					status: 200,
+				} );
+
+				await registry.dispatch( MODULES_ANALYTICS_4 ).submitChanges();
+
+				expect( fetchMock ).not.toHaveFetched( fpmSettingsEndpoint );
+			} );
+
+			it( 'should dismiss the FPM setup CTA banner when the FPM `isEnabled` setting is changed to `true`', async () => {
+				provideNotifications(
+					registry,
+					{
+						[ FPM_SETUP_CTA_BANNER_NOTIFICATION ]:
+							DEFAULT_NOTIFICATIONS[
+								FPM_SETUP_CTA_BANNER_NOTIFICATION
+							],
+					},
+					{ overwrite: true }
+				);
+
+				registry
+					.dispatch( CORE_SITE )
+					.receiveGetFirstPartyModeSettings( {
+						isEnabled: null,
+						isFPMHealthy: true,
+						isScriptAccessEnabled: true,
+					} );
+
+				registry.dispatch( CORE_USER ).receiveGetDismissedItems( [] );
+
+				const validSettings = {
+					accountID: fixtures.createProperty._accountID,
+					propertyID: fixtures.createProperty._id,
+					webDataStreamID: fixtures.createWebDataStream._id,
+				};
+
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.setSettings( validSettings );
+
+				fetchMock.postOnce( settingsEndpoint, {
+					body: validSettings,
+					status: 200,
+				} );
+
+				fetchMock.postOnce( fpmSettingsEndpoint, ( url, opts ) => {
+					const {
+						data: {
+							settings: { isEnabled },
+						},
+					} = JSON.parse( opts.body );
+
+					return {
+						body: {
+							isEnabled, // Return the `isEnabled` value passed to the API.
+							isFPMHealthy: true,
+							isScriptAccessEnabled: true,
+						},
+						status: 200,
+					};
+				} );
+
+				fetchMock.postOnce( dismissItemEndpoint, {
+					body: [ FPM_SETUP_CTA_BANNER_NOTIFICATION ],
 					status: 200,
 				} );
 
 				registry.dispatch( CORE_SITE ).setFirstPartyModeEnabled( true );
 				await registry.dispatch( MODULES_ANALYTICS_4 ).submitChanges();
 
-				expect( fetchMock ).toHaveFetched( fpmSettingsEndpoint, {
+				expect( fetchMock ).toHaveFetched( dismissItemEndpoint, {
 					body: {
 						data: {
-							settings: { isEnabled: true },
+							slug: FPM_SETUP_CTA_BANNER_NOTIFICATION,
+							expiration: 0,
 						},
 					},
 				} );
+				expect( fetchMock ).toHaveFetchedTimes( 3 );
+			} );
+			it( 'should handle an error when dismissing the FPM setup CTA banner', async () => {
+				provideNotifications(
+					registry,
+					{
+						[ FPM_SETUP_CTA_BANNER_NOTIFICATION ]:
+							DEFAULT_NOTIFICATIONS[
+								FPM_SETUP_CTA_BANNER_NOTIFICATION
+							],
+					},
+					{ overwrite: true }
+				);
+
+				registry
+					.dispatch( CORE_SITE )
+					.receiveGetFirstPartyModeSettings( {
+						isEnabled: null,
```

Comment:
Since we changed the `isEnabled` default value, let's update the mock as well.
```suggestion
						isEnabled: false,
```